apps/hvn/src/components/support/support-bank.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button } from '@fc/ui/base/button'
+import { Card, CardContent, CardHeader, CardTitle } from '@fc/ui/base/card'
+
+import { Copy } from 'lucide-react'
+
+export function SupportBank() {
+  const copyToClipboard = (text: string) => {
+    navigator.clipboard.writeText(text)
+  }
+
+  return (
+    <Card>
+      <CardHeader>
+        <CardTitle className="flex items-center gap-4">
+          <img alt="Stichting Logo" className="h-16 w-16 rounded-full object-cover shadow-md" src="/images/logo.png" />
+          <div>
+            <h3 className="text-xl font-bold">Harmonie van Nederland</h3>
+            <p className="text-sm text-muted-foreground">(Stichting Wees de Stem voor Vrijheid)</p>
```

Comment:
It is not necessary. Because this project is common. Freedom Combination and SANAT (There are two foundation in there)

But for donation info, we can add this sentence: 

This is a joint project of the SANAT and the Stg. Wees de stem voor vrijheid Foundation (Freedom Combination), but donations will be managed by the Freedom Combination.